test/verify/check-realms

```diff
@@ -259,6 +258,16 @@ class TestKerberos(MachineCase):
         self.configure_kerberos()
         self.machine.restart_cockpit()
 
+
+        # These only support the default keytab location
+        keytab = "/etc/cockpit/krb5.keytab"
+        if self.machine.image in ["rhel-7-4", "rhel-7-5"]:
+            keytab = "/etc/krb5.keytab"
+
+        # Pull keytab
+        self.machine.execute(["ipa-getkeytab","-q","-s","f0.cockpit.lan",
+                              "-p","HTTP/x0.cockpit.lan","-k",keytab])
```

Comment:
How does the cockpit-specific keytab  get onto the system here?

Reply:
the keytab variable is set to the cockpit specific one on the appropriate systems.